pkg/v1/providers/yttcb/clusterbootstrap.yaml

```diff
@@ -0,0 +1,218 @@
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:data", "data")
+#@ load("@ytt:ip", "ip")
+#@ load("lib/helpers.star", "get_tkr_version_from_tkr_name")
+
+#@ vsphereControlPlaneEndpointAsCidr = ""
```

Comment:
Question: Isn't there an existing lib function we can reuse? cc @anujc25

Reply:
this is the same as https://github.com/vmware-tanzu/tanzu-framework/blob/cad6db8b93af8786ac31e3457416f493e4aa9f95/pkg/v1/providers/ytt/02_addons/cpi/cpi_addon_data.lib.yaml#L10 we can consider refactor it into the lib.